hashids.cabal

```diff
@@ -1,36 +1,38 @@
--- Initial hashids.cabal generated by cabal init.  For further 
--- documentation, see http://haskell.org/cabal/users-guide/
-
 name:                hashids
 version:             1.0.2.4
 synopsis:            Hashids generates short, unique, non-sequential ids from numbers.
-description:         This is a Haskell port of the Hashids library. It is typically used to encode numbers to a format suitable to appear in visible places like urls. It converts numbers like 347 into strings like yr8, or a list of numbers like [27, 986] into 3kTMd. You can also decode those ids back. This is useful in bundling several parameters into one. 
+description:         This is a Haskell port of the Hashids library. It is typically used to encode numbers to a format suitable to appear in visible places like urls. It converts numbers like 347 into strings like yr8, or a list of numbers like [27, 986] into 3kTMd. You can also decode those ids back. This is useful in bundling several parameters into one.
 homepage:            http://hashids.org/
 license:             MIT
 license-file:        LICENSE
 author:              Johannes Hildén
 maintainer:          hildenjohannes@gmail.com
--- copyright:           
 category:            Web
 build-type:          Simple
-extra-source-files:  tests/*.txt
+extra-source-files:  test/testdata/*.txt
 cabal-version:       >=1.10
 
 library
   exposed-modules:     Web.Hashids
-  -- other-modules:       
-  other-extensions:    RecordWildCards
-  build-depends:       base ==4.*, bytestring, containers, split
```

Comment:
@intricate  I was rather thinking of adding some bounds on the other dependencies, because I sometimes get notifications from Stackage for other projects when something breaks due to a new version of a package. In recent projects I use something like `bytestring >=0.10 && <1` throughout. There are some different [schools of though](https://stackoverflow.com/questions/47795127/how-do-i-determine-reasonable-package-dependency-bounds-when-releasing-a-haskell) around how to do this, apparently.